examples/music_drop_fade.py

```diff
@@ -170,7 +170,8 @@ def main():
     pg.scrap.init()
     pg.SCRAP_TEXT = pg.scrap.get_types()[0]  # TODO remove when scrap module is fixed
     clipped = pg.scrap.get(pg.SCRAP_TEXT).decode(
-        "UTF-8")  # store the current text from the clipboard TODO remove decode
+        "UTF-8") 
```

Comment:
Another small nitpick, now that the comment is on the new line, `clipped = pg.scrap.get(pg.SCRAP_TEXT).decode("UTF-8")` can probably be on a single line rather than being broken like this

Reply:
Hi @ankith26 , yeah now i got it completely and made an another commit after updating it
please check now is it fine?

Reply:
Yes, it is looking good now